### PR TITLE
FINDING-10136 Phone icon added

### PR DESCRIPTION
**Pull request by untitledlt**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "dibs-vg",
-  "version": "1.28.0",
+  "version": "1.29.0",
```
Comment (hellatan) [inline package.json]: don't change the version until it's been merged

File: src/phone.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<svg version="1.1" class="dibsvg-svg dibsvg-phone" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (hellatan) [inline src/phone.svg]: i don't think you need the `x` and `y` props

File: package.json (hunk 1 above)
Comment (gedas-monginas) [inline package.json]: yes, should remove this commit

File: src/phone.svg [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+	 viewBox="0 0 250 250" enable-background="new 0 0 250 250" xml:space="preserve">
+<path fill="#231F20" d="M199.1,248.7l-0.2,0c-24.3-0.7-51.7-13.1-81.5-36.6c-20.9-16.5-35.2-32.6-37.1-34.7
```
Comment (gedas-monginas) [inline src/phone.svg]: and probably this `fill` isn't needed too

**Comment by untitledlt** [comment]
should be fine now

**Comment by gedas-monginas** [comment]
@hellatan not sure if you want to merge and publish, if not will do it on Monday

**Comment by hellatan** [comment]
published `dibs-vg@1.29.0`

**Pull request merged**